tests/e2e/default-mobile/old-navigation/old-navigation.wdio-spec.js

```diff
@@ -52,58 +51,51 @@ describe('Old Navigation', () => {
-    await messagesPage.sendMessage(message, person.name, person.phone );
+    const message = 'Navigations test';
+    await messagesPage.sendMessageOnMobile(message, person.name, person.phone );
     await messagesPage.openMessage(person._id);
 
-    const { name, phone } = await messagesPage.getMessageHeader();
+    const { name } = await commonPage.getHeaderTitleOnMobile();
     expect(name).to.equal(person.name);
-    expect(phone).to.equal(person.phone);
 
     const messages = await messagesPage.getAmountOfMessagesByPhone();
     const { content, state } = await messagesPage.getMessageContent(messages);
     expect(content).to.equal(message);
-    expect(state).to.equal('pending'); */
+    expect(state).to.equal('pending');
   });
 
   it('should navigate to the Task section and open the first task listed', async () => {
-    await browser.pause(100);
-    /* await oldNavigationPage.goToTasks();
+    await oldNavigationPage.goToTasks();
     await taskPage.openTaskById(
       pregnancyReport._id,
       '~pregnancy-danger-sign-follow-up~anc.pregnancy_danger_sign_followup'
     );
-    expect(await genericForm.getFormTitle()).to.equal('Pregnancy danger sign follow-up'); */
+    const { name } = await commonPage.getHeaderTitleOnMobile();
+    expect(name).to.equal('Pregnancy danger sign follow-up');
   });
 
   it('should navigate to the Reports section and open the first report listed', async () => {
-    await browser.pause(100);
-    /* await oldNavigationPage.goToReports();
+    await oldNavigationPage.goToReports();
     await reportsPage.openSelectedReport(await reportsPage.firstReport());
-    await waitForPageLoaded();
     const openReportInfo = await reportsPage.getOpenReportInfo();
     expect(openReportInfo.patientName).to.equal(person.name);
-    expect(openReportInfo.reportName).to.equal('Pregnancy registration'); */
+    expect(openReportInfo.reportName).to.equal('Pregnancy registration');
   });
 
   it('should navigate to the People section and open the created Health Center', async () => {
-    await browser.pause(100);
-    /* await oldNavigationPage.goToPeople();
+    await oldNavigationPage.goToPeople();
     await contactPage.selectLHSRowByText(healthCenter.name);
-    expect(await contactPage.getContactInfoName()).to.equal(healthCenter.name); */
+    expect(await contactPage.getContactInfoName()).to.equal(healthCenter.name);
   });
 
   it('should navigate to the Targets section, and open a target aggregate', async () => {
-    await browser.pause(100);
-    /* await oldNavigationPage.goToAnalytics();
+    await oldNavigationPage.goToAnalytics();
     await targetAggregatesPage.goToTargetAggregates(true);
-    await targetAggregatesPage.openTargetDetails(targetsConfig[0]); */
+    await targetAggregatesPage.openTargetDetails(targetsConfig[0]);
   });
 
   it('should successfully sync', async () => {
-    await browser.pause(100);
-    // await oldNavigationPage.sync();
+    await oldNavigationPage.goToPeople();
```

Comment:
```suggestion
```

Is there a specific reason why you are going to the people's tab first? I think that you can remove that line, because no matter where you are you can sync.

Reply:
@tatilepizs, the UI behaves differently for mobile screens. When you open a report or target in this instance the top bar is no longer visible, hence why I had to navigate back to the main tab.